data/file.go

```diff
@@ -56,8 +56,10 @@ func (f *File) IsPicture() bool {
 		if strings.HasSuffix(strings.ToLower(f.Name()), ending) {
 			// Then check MIME-type
 			if fileType, err := f.getFileContentType(); err == nil {
-				f.Close()
-				return strings.HasPrefix(fileType, "image/")
+				defer f.Close()
+				isPicture := strings.HasPrefix(fileType, "image/")
```

Comment:
It is common practice to create a variable for the return value (especially longer ones).